spec/presenters/features_spec.rb

```diff
@@ -5,22 +5,13 @@
 
 describe Features do
   describe '.wiki_ed?' do
-    context 'when the url is dashboard-testing.wikiedu.org' do
```

Comment:
This whole spec file can probably just be deleted, since the implementation is now trivial.